### PR TITLE
Import 5 ontologies into a compressed Neo4j database

### PR DESCRIPTION
**Pull request by dhimmel**
Imports DOID, UBERON, EFO, GO, and CMPO.

[`EBISPOT/OLS`](https://github.com/EBISPOT/OLS) is included as a submodule.

Uses git lfs to store the compressed database.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Biological ontologies as hetnets in Neo4j
+
+This repository creates a Neo4j database of prominent biological ontologies. The import is accomplished using the EBI's [`ols-neo4j-app`](https://github.com/EBISPOT/OLS/tree/master/ols-apps/ols-neo4j-app "EBISPOT/OLS on GitHub"). The compressed Neo4j database ([`hetontology.db.tar.xz`](database/hetontology.db.tar.xz)) can be easily deployed to a Neo4j instance to bring the convenience of the Cypher query language to ontologies.
```
Comment (cgreene) [inline README.md:3]: Might be helpful to specify if it includes the ontology terms _and_ the annotations or just the terms.

File: ontology-config/application-doid.properties [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# The ontology URI
```
Comment (cgreene) [inline ontology-config/application-doid.properties:1]: I'm ok with not including these comments everywhere, but the readme should probably point to this specific properties file as the example. The others (without comments) would be confusing to encounter first.

**Comment by cgreene** [comment]
Two small items that I noticed. Other than those :+1: - happy for you to fix them & merge. Thanks!

**Comment by cgreene** [comment]
Looks good to me! 👍

**Comment by dhimmel** [comment]
@cgreene, I addressed your comments. However, I'm going to hold off merging this pull request, since @simonjupp mentioned that the OLS team may be able to upload the database, updated daily, to their FTP site: https://github.com/EBISPOT/OLS/issues/104#issuecomment-235530608. This would supersede `database/hetontology.db.tar.xz`, the final output of this pull request.

So I forcefully reset the `master` branch to just the LICENSE commit and created a [`ols-neo4j-app` branch](https://github.com/greenelab/hetontology/tree/ols-neo4j-app) with this branch's code, since that's a more informative name than `dhimmel`. Therefore, if the FTP solution works out, we can close this pull request and preserve the code in the `ols-neo4j-app` branch but avoid introducing any of it into `master`.

@brettbj & @gwaygenomics, hence feel free not to review this PR.

**Pull request merged**